vuepress/docs/next/tutorials/customize-the-platform/pda-tutorial.md

```diff
@@ -1,278 +1,233 @@
 
-## Business Process Modeling Notation (bpmn)
+## Business Process Modeling Notation (BPMN)
 
 Business Process Model and Notation (BPMN) is a graphical representation
 for specifying business processes in a business process model.
 
-## Decision Model and Notation (dmn)
+## Decision Model and Notation (DMN)
 
-Decision Model and Notation is a standard published by the Object
-Management Group. It is a standard approach for describing and modeling
-repeatable decisions within organizations to ensure that decision models
-are interchangeable across organizations.
+Decision Model and Notation (DMN) is a standard published by the Object
+Management Group. It dictates a standardized approach for describing and modeling repeatable decisions within organizations to ensure that decision models are interchangeable across organizations.
 
-## Back End for Front End (BFF)
+## Backend for Frontend (BFF)
 
 A microservice architecture allows teams to iterate quickly and to
-develop technology to scale rapidly. The Backend for Frontend (BFF)
-architecture is a type of pattern built with microservices. The key
-component of this pattern is an application that connects the front-end
-of your application with the backend. This BFF Code Pattern will help
-you build that component according to IBM’s best practices.
+develop technology to scale rapidly. Backend for Frontend (BFF)
+is an architecture pattern built with microservices. The key
+component of this pattern is an application connecting the frontend
+of an application with the backend. The BFF Code Pattern helps to build that component according to IBM’s best practices.
 
 # PDA Micro Frontends
 
-This section provides an overview of each of the micro frontends (MFE)
-that are available as part of the Entando PDA plugin. In cases below
-where there are details specific to the PAM implementation they are
-provided for that MFE. When extending the integration layer to other
-engines or custom implementations it will be up to the integrating team
-to define behavior and datasource for each MFE.
+This section provides an overview of each micro frontend (MFE) that is available as part of the Entando PDA plugin. Details specific to the PAM implementation of an MFE are provided where appropriate. MFE behavior and datasources must be defined if the integration layer is extended to other
+engines or custom implementations.
 
 ## Task List
 
-The task list MFE provides a user with a list of visible tasks that are
-either assigned to that user or that are potentially ownable by that
-user. In the default implementation the visible tasks are limited to a
-single process instance. At configuration time the application designer
+The Task List MFE provides the user with a list of visible tasks that are
+either assigned to or potentially ownable by that user. In the default implementation, the visible tasks are limited to a
```

Comment:
owned?

Reply:
i struggled with that too... i kept "ownable" in case there's a distinction between the two, but ownable is awkward. and keeps triggering my autocorrect. @nshaw?

Reply:
@Lyd1aCla1r3 I think "either assigned to or could be claimed by that user." is more accurate. Not sure it's right for other BPM implementations but it works for PAM.